modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -583,11 +583,14 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
       throw new Error('Invalid user key - missing backupNShare');
     }
 
+    const derivationPath = txRequestResolved.unsignedTxs[0].derivationPath;
     const MPC = new Ecdsa();
-    const signingKey = MPC.keyCombine(userSigningMaterial.pShare, [
-      userSigningMaterial.bitgoNShare,
-      userSigningMaterial.backupNShare,
-    ]);
+    const keyDerive = MPC.keyDerive(
+      userSigningMaterial.pShare,
+      [userSigningMaterial.bitgoNShare, userSigningMaterial.backupNShare],
+      derivationPath
+    );
+    const signingKey = MPC.keyCombine(userSigningMaterial.pShare, [keyDerive.nShares[3], keyDerive.nShares[2]]);
```

Comment:
Not necessary for the user to call `keyCombine` here. In fact, the signingKey will not produce valid signing shares for the expected key. The user already has their signing material as `keyDerive.xShare`.